src/modules/common/App.jsx

```diff
@@ -30,12 +43,11 @@ export default function App({history}) {
     <ConnectedRouter history={history}>
       <div>
         <Nav>
-          <Link to={'/'}>Home</Link>
-          <Link to={'/health'}>Health</Link>
+          <Link className="home-link" to={'/'}><h2>OPENRECORD</h2></Link>
+          <Link className="contribute-link" to={'/'}><h2>ASK TO CONTRIBUTE</h2></Link>
```

Comment:
I realize this is a rough draft, but for the nav bar we should probably keep the links as <a> tags instead of title tags (unless there's an SEO reason for this?)